docs/community/com-query/grafana-connector.md

```diff
@@ -8,23 +8,30 @@ nav_order: 2
 
 # Grafana Connector
 
-Molecula provides a Grafana plugin, which can be used to interact with FeatureBase (4.0 or higher <!-- TODO is this correct? -->) to run queries and view visualizations of results. Note that this is unrelated to using Grafana for operational monitoring of the FeatureBase cluster. For details on that, see the [monitoring reference page](/docs/community/com-monitoring/old-monitoring)
+Featurebase provides a Grafana plugin, which can be used to interact with FeatureBase (4.0 or higher <!-- TODO is this correct? -->) to run queries and view visualizations of results. Note that this is unrelated to using Grafana for operational monitoring of the FeatureBase cluster. For details on that, see the [monitoring reference page](/docs/community/com-monitoring/old-monitoring)
 
 ## Setup
 
 ### Install Grafana
-For the best experience, [install](https://grafana.com/grafana/download) Grafana version 7 using the binary, not brew. The plugin requires version 7 or greater, but version 7 works best. For convenience, here are the commands for mac. For other platforms, check the link.
+For the best experience, [install](https://grafana.com/grafana/download) Grafana version 9 using the binary, not brew. The plugin requires version 7 or greater, but version 7 works best. For convenience, here are the commands for mac. For other platforms, check the link.
 
+For the best experience, [install](https://grafana.com/grafana/download) Grafana using the binary, not brew. The plugin requires version 7 or greater and works best on version 9.
+
+### Mac OS 
 ```
-curl -O https://dl.grafana.com/enterprise/release/grafana-enterprise-7.5.13.darwin-amd64.tar.gz
-tar -zxvf grafana-enterprise-7.5.13.darwin-amd64.tar.gz
+curl -O https://dl.grafana.com/oss/release/grafana-9.4.7.darwin-amd64.tar.gz
+tar -zxvf grafana-9.4.7.darwin-amd64.tar.gz
 ```
 
-### Install Plugin
-As of FeatureBase 4.3, a Grafana plugin archive should be included in your release archive. Unpack and note the location of the `/dist` folder.
-
+### Linux
+```
+wget https://dl.grafana.com/oss/release/grafana-9.4.7.linux-amd64.tar.gz
+tar -zxvf grafana-9.4.7.linux-amd64.tar.gz
+```
 
-In Grafana's configuration file `grafana-7-x.x/conf/defaults.ini` point the `plugins` field to the `/dist` folder.
+### Install Plugin
+Download the latest Featurebase release from [Featurebase](https://releases.molecula.cloud/). Unpack and locate the `/dist` directory. 
```

Comment:
You need login credentials to do this.
And if this is actually meant to be the releases of the FeatureBase Community application, there's existing instructions for different operating systems already written in the system:

* Install on Linux -- https://docs.featurebase.com/docs/community/com-install-linux/
* Install on MacOS -- https://docs.featurebase.com/docs/community/com-install-mac/
* Install on Windows (using Docker or WSL) -- https://docs.featurebase.com/docs/community/com-startup-connect/

In fact, this step would be better handled right at the top of the page under the standard "Before you begin" heading.  There's even a standard `/_include` shared file for this. For example:

```
## Before you begin
{% include /com-install/com-install-before-begin.md %}
```